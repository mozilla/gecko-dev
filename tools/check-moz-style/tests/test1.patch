Bad patch that doesn't diff any files
