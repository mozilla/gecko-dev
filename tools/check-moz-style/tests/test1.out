patch:0:  Patch does not appear to diff against any file.  [patch/notempty] [3]
